<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportCsvDialog</class>
 <widget class="QDialog" name="ExportCsvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export data as CSV</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTable">
       <property name="text">
        <string>&amp;Table(s)</string>
       </property>
       <property name="buddy">
        <cstring>listTables</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QListWidget" name="listTables">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>0</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHeader">
       <property name="text">
        <string>&amp;Column names in first line</string>
       </property>
       <property name="buddy">
        <cstring>checkHeader</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkHeader">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFieldSeparator">
       <property name="text">
        <string>Field &amp;separator</string>
       </property>
       <property name="buddy">
        <cstring>comboFieldSeparator</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboFieldSeparator">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>,</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>|</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editCustomSeparator">
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelQuoteCharacter">
       <property name="text">
        <string>&amp;Quote character</string>
       </property>
       <property name="buddy">
        <cstring>comboQuoteCharacter</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboQuoteCharacter">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>'</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editCustomQuote">
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelNewLineCharacters">
       <property name="text">
        <string>New line characters</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
       <item>
        <widget class="QComboBox" name="comboNewLineString">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Windows: CR+LF (\r\n)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unix: LF (\n)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editCustomNewLine">
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listTables</tabstop>
  <tabstop>checkHeader</tabstop>
  <tabstop>comboFieldSeparator</tabstop>
  <tabstop>editCustomSeparator</tabstop>
  <tabstop>comboQuoteCharacter</tabstop>
  <tabstop>editCustomQuote</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportCsvDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportCsvDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboFieldSeparator</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ExportCsvDialog</receiver>
   <slot>showCustomCharEdits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboQuoteCharacter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ExportCsvDialog</receiver>
   <slot>showCustomCharEdits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboNewLineString</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ExportCsvDialog</receiver>
   <slot>showCustomCharEdits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCustomCharEdits()</slot>
 </slots>
</ui>
